src/hotspot/share/runtime/registerMap.hpp

```diff
@@ -153,8 +153,10 @@ class RegisterMap : public StackObj {
   const RegisterMap* as_RegisterMap() const { return this; }
   RegisterMap* as_RegisterMap() { return this; }
 
+#ifndef PRODUCT
   void print_on(outputStream* st) const;
   void print() const;
+#endif
```

Comment:
They can be merged into a single MACRO with line 165.

Reply:
Thanks, updated.